Rakefile

```diff
@@ -13,8 +13,6 @@ Rake::TestTask.new(:test) do |t|
   t.libs << 'lib'
   t.libs << 'test'
   t.pattern = ENV['TEST_PATTERN'] || 'test/**/*_test.rb'
-  t.verbose = ENV['TEST_VERBOSE'] == '1'
-  t.warning = true
```

Comment:
I think this is a bad idea. Warnings are good to come across things we need to fix